frontend/src/components/EventList/EventMessage.tsx

```diff
@@ -43,7 +43,7 @@ export function EventMessage({
     case EventType.device_state:
       message = (
         <>
-          <b> {item.target && serviceName !== device?.name ? serviceName : device?.name} </b>
+          <b> {item.target && serviceName !== device?.name ? serviceName : device?.name + ' [ ' + device?.id + ' ] ' }  </b>
```

Comment:
```suggestion
import { addDeviceName } from 'nameHelper.ts' // need real path

let name = device?.id
if (device?.name) name = device.name
if (serviceName) name = addDeviceName(name, serviceName)

...

          <b> {item.target && name}  </b>
```

What about something like this? We want to show device name if available and service name if available. and only if those don't exist then show device id.